protoc-gen-rust-grpc/README.md

```diff
@@ -0,0 +1,64 @@
+## Build
+
+To build the Rust gRPC code generator plugin:
+
+```sh
+bazel build //src:protoc-gen-rust-grpc
+```
+
+
+## Usage Example
+
+```sh
+# Build the plugin
+bazel build //src:protoc-gen-rust-grpc
+
+# Set the plugin path
+PLUGIN_PATH="$(pwd)/bazel-bin/src/protoc-gen-rust-grpc"
+
+# Run protoc with the Rust and Rust gRPC plugins
+protoc \
```

Comment:
I think we should probably advise users to use `grpc_build::CodeGen` and/or `protobuf_codegen::CodeGen` instead of directly invoking protoc. If you directly invoke protoc and check in the generated code, then it's at risk of becoming stale if/when you upgrade your protobuf dependency.